package.json

```diff
@@ -14,9 +14,9 @@
   },
   "license": "BSD-3-Clause",
   "scripts": {
-    "test-node": "mocha --recursive -R dot test/**-test.js",
+    "test-node": "mocha --recursive -R dot 'test/**/*-test.js'",
```

Comment:
This probably won't fly with Windows, if I am not mistaken. I think it needs to use escaped double quotes.